tests/UserTest.php

```diff
@@ -260,6 +260,15 @@ public function testGet()
         $this->assertEqualsFixture('get_user', $response->json());
     }
 
+    public function testGetWrongUrl()
```

Comment:
What happens if delete does not have wereNumber(id)?
Please add tests for delete and put also

```suggestion
    public function testGetIdParamAsString()
```